Gemfile

```diff
@@ -1,6 +1,9 @@
 source 'https://rubygems.org' do
-  group :development do
+  gem 'spoon' # Added because jruby seems to want it?
```

Comment:
I think that simplecov use pid to generate coverage results.
Maybe, JRuby need 'spoon gem' to make process through ffi 
But CRuby doesn't need 'spoon gem' because CRuby has a core extension with c lang.